Add builmenlabo custom node entry

comnote-max opened the pull request:
# 追加内容
- **name**: builmenlabo  
- **repo**: comnote-max/builmenlabo  
- **path**: comfyui-builmenlabo  
- **description**: builmenlabo 用のユーティリティ系カスタムノード集  
- **author**: comnote-max  
- **category**: utilities  
- **tags**: builmenlabo, custom-node  

# 動作確認環境
- Windows 11  
- ComfyUI 0.3.35  
- ComfyUI‑Manager V3.32.8  

# ライセンス
MIT

ご確認のほどよろしくお願いいたします。

comnote-max committed:
Add files via upload

comnote-max committed:
Add files via upload

comnote-max commented:
@Comfy-Org/maintainers  
お手すきの際に #2020 のワークフロー承認（Approve and run）をお願いします🙏

@Comfy-Org/maintainers  
Could you please approve and run the workflow for PR #2020 when you have a moment?  
Thank you! 🙏

ltdrdata commented:
To make the repository installable via git clone, an `__init__.py` file must be located at the root of the repo.

comnote-max commented:
ご指摘ありがとうございます。
リポジトリのルートに `__init__.py` を追加し、不要になった `comfyui‑builmenlabo` フォルダを削除しました。
お手すきの際にご確認いただけますと幸いです！

---

Thank you for your feedback.
I have added `__init__.py` to the repository root and removed the obsolete `comfyui‑builmenlabo` folder.
Please review the changes at your convenience.

ltdrdata committed:
Delete manager_registration.json
